app/sliders/pn_app.py

```diff
@@ -22,40 +25,128 @@
     "y": [6, 7, 2, 4, 5]
 }
 
+interactive_line_data = {
```

Comment:
This was dummy data that anyway gets erased, right? Just for good measure I would always try to avoid german keys or ids because it hurts a little in the eye😛

Reply:
Genau, all of this is dummy data that will be erased later